### ISSUE TITLE
Server Can't keep up! 1.5.8

### ISSUE THREAD

**Issue by McKocian** [MyM-ModpackTeam/GalacticScience#108]
My problem is that I run a server on a server computer, and when my friend and I try to make a lot of rf with the advanced generators fueled with syngas, the server can't keep up, We mad a treefarm and automate it with the ender io conduits and a crafter, like everyone on youtube. We've got a chopper befor but it didn't make lag. What sould we do now? I remove our Dimensional anchors too, but the ms keep high.

**Comment by Bennyboy1695** [comment]
Galactic Science is a big pack so needs quire a lot of stuff dedicated to it. What size is the server your currently trying to run it on ?
It works best with 3 - 4gb ram dedicated to it.

**Comment by McKocian** [comment]
Yes we dedicated 4gb ram just for this server. But I try to run the server on my notebook (it has just 2gb free ram for the server), and It gets the exactly same problem :\

**Comment by SirWilli** [comment]
Check the tps of your server, use opis other similiar tools to profile it.

**Comment by McKocian** [comment]
We found the source of the high ms. We've been trying to make a base on the Venus and the chopper and the tree popuation, nearly 40 trees, killed the server whenever a sapling grown up to a tree. We didn't think about it earlier caus we do the same porcess to the advancedgenerators on the Moon, and we moved it to the Venus when we think that would be good to get more power. So maybe this comment wold help you guys to figure out how could be possible to make massive wood generate on Venus without lag.

Sorry for my bad grammer, I eloped from the English lagnuage.
